src/lib/radio/radio.ts

```diff
@@ -149,6 +152,18 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
     }
   }
 
+  @Input()
+  get required(){
+    return this._required;
+  }
+
+  set required(required: boolean) {
+    this._required = required;
+
+    if (this._radios) {
+      this._radios.forEach(radio => radio.required = this.required);
+    }
```

Comment:
I found the `ng-reflect-required` thingy. I used the `required="true"` without the brackets.